libs/testserver/server.go

```diff
@@ -19,7 +19,7 @@ import (
 
 type Server struct {
 	*httptest.Server
-	Router *mux.Router
+	router *mux.Router
```

Comment:
this and other changes in this file don't seem necessary, could you clean up?

Reply:
This is intentional, we don't use the Router outside this package. Better to keep it private.

Reply:
reverted